test/factory.ts

```diff
@@ -25,16 +27,57 @@ describe('factory', () => {
     sandbox.restore();
   });
 
+  describe('gitHubInstance', () => {
+    const owner = 'google';
+    const repo = 'cloud';
+    const repoUrls = [
+      `${owner}/${repo}`,
+      `${owner}/${repo}`,
+      `https://github.com/${owner}/${repo}.git`,
```

Comment:
I appreciate these tests, and that we're so forgiving with the input.

Reply:
oops, I have the same input listed twice. I was just trying to copy some examples from https://github.com/repo-utils/parse-github-repo-url#features